### DISCUSSION TITLE
TS1206: Decorators are not valid here. Como resolver?

### DISCUSSION THREAD

**Discussion by mrsaoc** [balta-io/forum#449] [Tira Dúvidas] [answered]
Olá, estou iniciando no TS (curso 7181, aula:Form Básico) e tenho uma dúvida, quando coloco um decorator aparece um pop-up na tela com o seguinte erro: TS1206: Decorators are not valid here. Gostaria mto da ajuda de vocês para eu continuar o seguimento com o curso! Obrigado.

![image](https://github.com/balta-io/forum/assets/61404851/3c93d9b0-892f-4180-8ce0-caf2c438472e)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @marcosvmg, está usando as mesmas versões dos pacotes e TS do curso?

**Reply by mrsaoc** [reply]
bom dia André! Não estava, no Natal reinstalei e agora tá funcionando certinho, muito obrigado.

**Reply by andrebaltieri** [reply]
Perfeito!! Bons estudos 🚀
